test/modules/TestResultCoordinator/ITestOperationResultStorage.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Storage;
+    public interface ITestOperationResultStorage
+    {
+        ISequentialStore<TestOperationResult> GetStoreFromSource(string source);
+
+        Task<bool> AddResultAsync(TestOperationResult testOperationResult);
```

Comment:
How do you retrieve the results?

Reply:
We use GetStoreFromSource with ReportMetadata, which contains the following info about the report: ExpectedSource, ActualSource, TestReportType, and TestOperationResultType. 

For each ReportType we have, we will have a new testResultReportGenerator that will generate a report for that Type.

GetStoreFromSource is used in TestReportGeneratorFactory to create TestResultReportGenerator's (currently only CountingReportGenerator)